src/components/modes/student/StudentView.js

```diff
@@ -28,75 +27,72 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-// const timerType = "backward"
-// const startValue = timerType === "forward" ? 0 : 4800000
+const GridItem = ({ timeValue, timeUnit }) => (
+  <Grid item xs={3} style={{ width: '100%' }}>
```

Comment:
The style tag should be avoided. Instead, pass a `className={classes.gridItem}` and add the styles in the styles object.